linux_os/guide/system/accounts/accounts-session/root_paths/accounts_root_path_dirs_no_write/ansible/shared.yml

```diff
@@ -3,27 +3,27 @@
 # strategy = restrict
 # complexity = low
 # disruption = medium
-- name: "Fail if user is not root"
+- name: "Print error message if user is not root"
   fail:
     msg: 'Root account required to read root $PATH'
   when: ansible_user != "root"
+  ignore_errors: true
```

Comment:
Leaving a comment for future documentation:  Idea is for playbook not to fail entirely, rather start skipping remediations if/when needed and continue hardening.